ansible_base/management/commands/update_resources.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand
+
+from ansible_base.models import Permission, Resource, ResourceType
+
+
+class Command(BaseCommand):
+    help = "Reset resource index"
+
+    def handle(self, *args, **options):
+        ResourceType.objects.all().delete()
+
+        ResourceType.update_resource_types_from_registry()
+        Permission.update_permissions()
+        Resource.update_index()
```

Comment:
Why would we want to reset this during normal operation?

Reply:
Oh, I need to get rid of this. I added that for testing. It's horribly out of date.